Tests/TestCodeGenConfigurations/EmbeddedInTarget-InSchemaModule/apollo-codegen-config.json

```diff
@@ -17,7 +17,7 @@
   },
   "input" : {
     "operationSearchPaths" : [
-      "../../../Sources/AnimalKingdomAPI/animalkingdom-graphql/**/*.graphql"
+      "../../../Sources/AnimalKingdomAPI/animalkingdom-graphql/*.graphql"
```

Comment:
Excluding CCN operations from test configurations, also to be finally handled in #3151.